src/components/queue/QueueList/QueueList.tsx

```diff
@@ -64,9 +84,21 @@ const QueueList: FC<QueueListProps> = ({queue, showCompletedTickets}) => {
                     Queue
                 </Typography>
                 {!queue.isCutOff && !queueEnded && !inQueue && !isTA(queue.course.id) &&
-                    <Button variant="contained" onClick={() => setCreateTicketDialog(true)}>
-                        Join Queue
-                    </Button>}
+                    <div>
```

Comment:
Logic to handle the tooltip - I had to wrape the button in a div element for the tooltip to display over the disabled button. Also, I was having difficulty cleanly nesting it so I made a parent div and then another conditional render statement. This could likely be made cleaner but it is functional and it displays the message to the user if the button is disabled.